readme.md

```diff
@@ -609,6 +609,7 @@
 - [Board Games](https://github.com/edm00se/awesome-board-games#readme) - Table-top gaming fun for all.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
 - [Parasite](https://github.com/ecohealthalliance/awesome-parasite#readme) - Parasites and host-pathogen interactions.
+- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.
```

Comment:
This should be added to the bottom of the section.

Reply:
that was the bottom when i initially opened the PR back in january. 😛 
in any case, it's fixed now.